sycl/test-e2e/Matrix/joint_matrix_gemm_cuda.hpp

```diff
@@ -174,27 +200,32 @@ void test(queue &q) {
               }
 
               joint_matrix_mad(sg, sub_d, sub_a, sub_b, sub_c);
+              joint_matrix_copy(sg, sub_d, sub_c);
```

Comment:
I'm wondering, how this test worked before without this copy?

Reply:
When I last changed the test to support the case that D matrix is a different type than the C matrix (the only case the cuda backends supports this is for C/D = fp32/fp16), I made `constexpr int SUB_TILES_K = 1`. If `SUB_TILES_K` were greater than 1, the test would have failed. I've made this change here to allow SUB_TILES_K greater than K, mainly for illustrative purposes, so people may look to this example to understand how to deal with strides/load_strides.

Reply:
Makes sense, thank you for explaining.